utils/auto_dockerfile_update/update_dockerfiles.py

```diff
@@ -156,7 +156,7 @@ def run_lock(base_path_docker: str, pipfile_or_pyproject_path: str) -> bool:
         if "Pipfile" in pipfile_or_pyproject_path:
             # waits for the process to end.
             completed_process = subprocess.run(
-                ["pipenv", "lock", "--keep-outdated"],
+                ["pipenv", "lock"],
```

Comment:
wont this update all the dependencies? Why do we want to do that?

Reply:
Yes.
Because there is no other solution... those arguments are not supported and I checked this on 4-5 dockers and their build passes ok even if all the dependencies are updated.

Reply:
there will be an issue for dependencies we specifically have ignored in dependabot config, and major updates will happen without review